internal/service/cluster/collector/collector_k8s.go

```diff
@@ -18,9 +18,9 @@ import (
 )
 
 const vendorLabelName = "vendor"
-const openshiftVersionLabelName = "openshiftVersion"
-const clusterIDLabelName = "clusterID"
-const localClusterLabelName = "local-cluster"
+const OpenshiftVersionLabelName = "openshiftVersion"
```

Comment:
we shouldn't take this from here. Nothing wrong to have our constants. But if you want something common for both. Use internal/controllers/utils/constants.go. We are adding common constants there. Some at the end of the file

Reply:
moved to utils